packages/data-stores/src/reader/types/index.ts

```diff
@@ -1,9 +1,12 @@
 export type EmailFormatType = 'html' | 'text';
 
+export type DeliveryWindowDayType = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+export type DeliveryWindowHourType = 0 | 2 | 4 | 6 | 8 | 10 | 12 | 14 | 16 | 18 | 20 | 22;
+
 export type SubscriptionManagerUserSettings = Partial< {
 	mail_option: EmailFormatType;
-	delivery_day: number; // 0-6, 0 is Sunday
-	delivery_hour: number; // 0-23, 0 is midnight
+	delivery_day: DeliveryWindowDayType; // 0-6, 0 is Sunday
+	delivery_hour: DeliveryWindowHourType; // 0-23, 0 is midnight
```

Comment:
Now that we have the types, we can remove the comment
```suggestion
	delivery_hour: DeliveryWindowHourType;

Reply:
That's true! Great catch. Thanks! Going to fix it now.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/74763/commits/38055d058bec70ae3cd771d41d949174f3781359.